instrumentation/aws-sdk/aws-sdk-2.2/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/awssdk/v2_2/TracingExecutionInterceptor.java

```diff
@@ -41,10 +39,6 @@
  */
 public class TracingExecutionInterceptor implements ExecutionInterceptor {
 
-  public static class ScopeHolder {
-    public static final ThreadLocal<Scope> CURRENT = new ThreadLocal<>();
-  }
-
   private final ExecutionInterceptor delegate;
 
   public TracingExecutionInterceptor() {
```

Comment:
I think after #1643 I may try to remove this class, instead just service loading the shaded library instrumentation.